Feliz.UseMediaQuery/Feliz.UseMediaQuery.fsproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <PackageVersion>0.1</PackageVersion>
+        <Authors>Jonas Bösch, Zaid Ajaj</Authors>
+        <Description>useMediaQuery hooks to build responsive websites </Description>
+        <PackageLicenseUrl>https://github.com/Zaid-Ajaj/Feliz/blob/master/LICENSE</PackageLicenseUrl>
+        <RepositoryUrl>https://github.com/Zaid-Ajaj/Feliz</RepositoryUrl>
+        <PackageTags>fsharp;fable;react;html;feliz</PackageTags>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Content Include="*.fsproj; *.fs; *.js;" PackagePath="fable\" />
+        <Compile Include="UseMediaQuery.fs" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Fable.Browser.MediaQueryList" Version="1.1.0" />
+      <PackageReference Include="Fable.React" Version="6.2.0" />
+      <PackageReference Include="Feliz" Version="1.6.0" />
```

Comment:
Maybe it is better to make it reference `Feliz` as a project rather the external nuget, this way every time you push a new version, it will use latest Feliz

Reply:
Ok, fixed